app/dashboard/models.py

```diff
@@ -252,6 +254,30 @@ def fetch_issue_description(self):
         if body:
             self.issue_description = body
 
+    def fetch_issue_comments(self, save=True):
+        from urlparse import urlsplit
+        from app.github import get_issue_comments
+
+        if self.github_url.lower()[:19] != 'https://github.com/':
+            return
+
+        parsed_url = urlsplit(self.github_url)
+        try:
+            github_user, github_repo, _, github_issue = parsed_url.path.split('/')[1:5]
+        except ValueError:
+            # TODO: update print statements to logger
+            print('Invalid github url')
```

Comment:
😉

Reply:
i wonder if it's worth logging and error here

Reply:
probably, the repo isn't setup for using the built-in logger that I saw -- easy enough to add but was aiming for a clean PR to make it easy to review

Reply:
there is _some_ of the built in logging system being done in `notifications.py` ... not sure if that qualifies or helps or not

Reply:
want me to throw a logging config into settings?

pretty much copy and paste, can dial it in later but will open up using `logging.getLogger`